test/dynamo/test_functions.py

```diff
@@ -2126,7 +2157,7 @@ def fn(x, y):
             else:
                 return x + y
 
-        fn_opt = torch.compile(backend="eager", fullgraph=True, dynamic=True)(fn)
+        fn_opt = torch.compile(backend="eager", fullgraph=False, dynamic=True)(fn)
```

Comment:
Do we have a reason to disable fullgraph mode?

Reply:
@yanboliang Triggers failing tests otherwise (since we have now have a graph break on the `is_`

Reply:
We shouldn't regress on ```is_```.